dimensions:
  d_eq: 0.5        # Equilibrium distance [m]
  radius: 0.2      # Radius of the robots [m]
  max_vis_objs: 2  # Max distance of visibility [m]

gains:
  k_r: 4.0      # Inter-robot force gain
  k_o: 0.7      # Obstacle avoidance force gain
  k_m: 0.8      # Migration force gain
  k_al: 1       # Alignment force gain 
  k_l: 0.05     # Linear Velocity gain
  k_a: 5.0      # Angular Velocity gain

bounds:
  v_min: 0
  v_max: 0.5
  omega_min: -10
  omega_max: 10
  a_max: 2 
  force_max: 4
  angle_offset_max: 30  # in degrees
  
agent:
  is_omnidirectional: false
  is_influenced: false
  num_options: 4
  k: 0.6
  h: 0.2

# Ostacolo sta sulla diagonale a 4 mattonelle 